.github/workflows/ci-test.yml

```
@@ -0,0 +1,42 @@
+name: CI-Test
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+    types: [opened, synchronize]
+
+  jobs:
```

actionlint

unknown Webhook event "jobs". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)